src/Mod/Fem/Gui/ViewProviderFemPostPipeline.cpp

```diff
@@ -137,6 +137,8 @@ void ViewProviderFemPostPipeline::transformFields()
     if (!data || !data->IsA("vtkDataSet"))
         return;
 
+    return;
```

Comment:
What's the point of the "return" statement. Doesn't look like it makes much sense.

Reply:
The code below the return was never tested since I spent all the time to the exposure to Python.
Therefore the PR was a Draft all the time.

(Today it turned out that the reason for the PR is no longer there - the behavior of FC 0.20 is correct: https://forum.freecadweb.org/viewtopic.php?p=614282#p614282)